test/factories/study.rb

```diff
@@ -25,6 +25,7 @@
     # create a study but mark as detached, so a Terra workspace is not created
     factory :detached_study do
       detached { true }
+      bucket_id { SecureRandom.alphanumeric(16) }
```

Comment:
can you add a comment here for why it's better to have a filler value here than leaving it null?

Reply:
Sure - it was throwing an error in one of my new tests because the mock was trying to access that value, and the `nil` value cause a `MockExpectationError`.

Reply:
7b97748